docs/source/modin/index.rst

```diff
@@ -18,4 +18,5 @@ For your convenience, here is all the :doc:`Supported APIs <supported/index>`
     groupby
     resampling
     numpy
-    All supported APIs <supported/index>
\ No newline at end of file
+    All supported APIs <supported/index>
+    performance
```

Comment:
uppercase?

Reply:
This is lowercase (as are the others) since its actually just letting sphinx know the name of the page. In the index, it is appropriately capitalized, since I believe it grabs the title of the page from the page itself. Adding a screenshot below for verification:
![image](https://github.com/snowflakedb/snowpark-python/assets/149019786/9d4c5fc3-5909-4548-9c36-b6244c0a7522)